Assets/Scripts/GridManagerModel.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+
+public class GridManagerModel
+{
+    private GameSettings _gameSettings;
+    private GameObject[,] _gridCellsArray;
+    private GridCell gridCellScript;
+    private GridManagerViewer _gridManager;
+    private CandyPool _candyPool;
+    public GridManagerModel (GameSettings gameSettings, GameObject[,] gridCellsArray, GridManagerViewer gridManager, CandyPool candyPool)
+    {
+        _gameSettings = gameSettings;
+        _gridCellsArray = gridCellsArray;
+        _gridManager = gridManager;
+        _candyPool = candyPool;
+    }
+
+    public void PopulateBackdropGrid(Vector2 firstTilePos, GameObject gridCellGO, GameObject[,] candiesArray)
+    {
+        for (int i = 0; i < _gameSettings.tilesNumberI; i++)
+        {
+            for (int j = 0; j < _gameSettings.tilesNumberJ; j++)
+            {
+                Vector2 position = new Vector2(firstTilePos.x + j * _gameSettings.tileSize, firstTilePos.y - i * _gameSettings.tileSize);
+                gridCellGO = _gridManager.InstantiateGridCell();
+                gridCellGO.transform.position = position;
+                gridCellGO.transform.localScale = new Vector3(_gameSettings.tileSize, _gameSettings.tileSize, 1);
```

Comment:
Setting position and local scale is something that should be in the `View` not in the model.